docs/ref/models/instances.txt

```diff
@@ -528,7 +528,7 @@ Updating attributes based on existing fields
 --------------------------------------------
 
 Sometimes you'll need to perform a simple arithmetic task on a field, such
-as incrementing or decrementing the current value. The obvious way to
+as incrementing or decrementing the current value. The simplest way to
```

Comment:
`simplest` was one of words removed in #11482. I rephrased this.

Reply:
Yeah good point. I think here it was still warranted - it's not saying it's necessarily "simple and obvious" but just that it's the "simplest" option, even if it's not (yet?) obvious or simple for the reader. I think your rephrasing is fine, although it has possibly lost the "signposting" that this first option is not necessarily the best.

Reply:
Maybe _"A common way of achieving this is ..."_ :thinking: